### DISCUSSION TITLE
Add support for directus command in docker image

### DISCUSSION THREAD

**Discussion by danielgroen** [directus/directus#23225] [Feature Requests]
### Summary

if i want to run any docker commands like this: `cli.js start`.

However it would be nice to create a alias named directus.
Especially if you have a dockerfile where you need a custom CMD.

### Basic Example

Instead of `cli start` you can just write: `directus start`

### Motivation

it would be nice because then your commands look just as regular directus commands.
Also when you log in in the docker container you'll always have the directus command available from any directory

### Detailed Design

```
RUN ln -s /directus/cli.js /bin/directus
```

### Requirements List

N/A

### Drawbacks

N/A

### Alternatives

```
RUN ln -s /directus/cli.js /usr/local/bin
```

### Adoption Strategy

N/A

**Comment by rijkvanzanten** [comment]
Wanna open a PR for this? Feels like this should be possible by creating an alias in the Dockerfile I think? 🤔
